process_data.sh

```diff
@@ -250,10 +250,18 @@ sct_process_segmentation -i ${file_t2s_seg}.nii.gz -angle-corr 0 -vert 3:4 -vert
 
 # DWI
 # ------------------------------------------------------------------------------
-file_dwi="${SUBJECT}_dwi"
 cd ../dwi
+if [ -f ${SUBJECT}_dwi.nii.gz ];then
+  file_dwi="${SUBJECT}_dwi"
+else
+  file_dwi="${SUBJECT}_run-01_dwi"
+fi
 # If there is an additional b=0 scan, add it to the main DWI data
-concatenate_b0_and_dwi "${SUBJECT}_acq-b0_dwi" $file_dwi
+if [ -f ${SUBJECT}_acq-b0_dwi.nii.gz ];then
+  concatenate_b0_and_dwi "${SUBJECT}_acq-b0_dwi" $file_dwi
+else
+  concatenate_b0_and_dwi "${SUBJECT}_acq-b0_run-01_dwi" $file_dwi # RL-20211208: I am not 100% sure, if this is the right naming format for $1
```

Comment:
What do you mean by '$1'?

Reply:
I meant: "${SUBJECT}_acq-b0_run-01_dwi" If it is accurate or if e.q. the right strin is: "${SUBJECT}_run-01_acq-b0_dwi"

Reply:
it should be `"${SUBJECT}_run-01_acq-b0_dwi"`

Reply:
Repaired in commit b3075a7